### ISSUE TITLE
Database group name

### ISSUE THREAD

**Issue by TdroL** [creatoro/jelly#10]
In some cases there won't be group 'default' in config file (eg. there will be insdtead 'production', 'development', etc.) -- Jelly should get group name from Database::$default (I assume that this variable have correct name).

Jelly_Core_Meta needs changes:
    protected $_db = NULL;
        (...)
    public function db($value = NULL)
    {
        if (func_num_args() !== 0)
        {
            return $this->set('db', $value);
        }

```
    return isset($this->_db) ? $this->_db : Database::$default;
}
```

**Comment by creatoro** [comment]
Thanks for this. The builder has hard-coded db name as well.

**Comment by creatoro** [comment]
Closed by 963c997b4bf76af7280f2f89265ca571dfefd100, thanks TdroL.
